src/main/routes/admin.ts

```diff
@@ -0,0 +1,17 @@
+import { UserLevel } from '../types/user-level';
+import { SessionUser } from '../services/session-user/session-user';
+
+import { Application } from 'express';
+import { requiresAuth } from 'express-openid-connect';
+
+export default function (app: Application): void {
+  app.get('/admin', requiresAuth(), async (req, res) => {
+    const userRole = SessionUser.getLoggedInUserProfile(req).app_access[0]?.role.name;
+    if (userRole === UserLevel.SUPER_USER) {
```

Comment:
The app access list is not sorted so SUPER USER role might not be the first one in the list:
```suggestion
    if (SessionUser.getLoggedInUserProfile(req).app_access.filter((role) => role.role.name === UserLevel.SUPER_USER).length > 0) {
```